test/fakefs_test.rb

```diff
@@ -1292,6 +1292,7 @@ def test_should_set_pos_with_tell_method
   ]
 
   (RealFile.instance_methods - OMITTED_FILE_METHODS).each do |method_name|
+    next if defined?(JRUBY_VERSION) && [:ttymode, :ttymode_yield, :to_channel, :to_outputstream, :to_inputstream].include?(method_name)
```

Comment:
Should we instead change the `OMITTED_FILE_METHODS` constant to an `omitted_mri_file_methods`, add an `omitted_jruby_file_methods` and put the `if defined?` logic into an `omitted_file_methods` that can be used for the subtraction in line 1294? I think that would be cleaner.